### DISCUSSION TITLE
How to deform mesh by control points?

### DISCUSSION THREAD

**Discussion by zhang-qiang-github** [CGAL/cgal#7646] [Q&A] [answered]
I have a face mesh and the key point of nose. I want to make the nose bigger. How can I deform this mesh with CGAL?

Any suggestion is appreciated~~~

**Comment by janetournois** [comment, marked as answer]
You can use the [Surface Mesh Deformation package](https://doc.cgal.org/latest/Surface_mesh_deformation/index.html)

**Reply by zhang-qiang-github** [reply]
Thanks a lot~~~
